core/gui/src/app/dashboard/component/admin/settings/admin-settings.component.ts

```diff
@@ -204,8 +210,28 @@ export class AdminSettingsComponent implements OnInit {
   }
 
   saveDatasetSettings(): void {
-    if (this.maxFileSizeMB < 1 || this.maxConcurrentChunks < 1 || this.chunkSizeMB < 1) {
-      this.message.info("Value must be at least 1.");
+    if (this.maxFileSizeMB < 1 || this.maxFileSizeMB > this.MAX_FILE_SIZE_MB) {
```

Comment:
This is not necessary. NzMin/NzMax and this are both frontend check, they are effectively the same.